diff -Naur i3-4.7-orig/src/manage.c i3-4.7/src/manage.c
--- i3-4.7-orig/src/manage.c	2013-12-22 12:12:41.000000000 -0800
+++ i3-4.7/src/manage.c	2014-01-03 10:50:11.103285857 -0800
@@ -412,6 +412,12 @@
                 transient_win = next_transient->window;
             }
         }
+    } else {
+        if (config.popup_during_fullscreen == PDF_LEAVE_FULLSCREEN &&
+            fs != NULL) {
+            LOG("The window is not transient, but there is a fullscreen window: leaving fullscreen mode\n");
+            con_toggle_fullscreen(fs, CF_OUTPUT);
+        }
     }
 
     /* dock clients cannot be floating, that makes no sense */
